xikolo-common/Data/Helper/FetchRequests/AnnouncementHelper+FetchRequests.swift

```diff
@@ -21,10 +21,18 @@ extension AnnouncementHelper {
 
         static var unreadAnnouncements: NSFetchRequest<Announcement> {
             let request: NSFetchRequest<Announcement> = Announcement.fetchRequest()
+            let dateSort = NSSortDescriptor(key: "publishedAt", ascending: false)
+            request.sortDescriptors = [dateSort]
```

Comment:
Why do we need this?

Reply:
I forgot that we only use this for the number. I thought we already had a filter for unread announcements and they should have been ordered by date.